### ISSUE TITLE
Can't support `num` type ?

### ISSUE THREAD

**Issue by taosimple** [ethanblake4/dart_eval#98]
```
num ADD(num a, num b){
  return a + b;
}
```
eval codes above will throw error:
```
dart_eval runtime exception: type 'int' is not a subtype of type '$Value' in type cast
#0      Unbox.run (package:dart_eval/src/eval/runtime/ops/primitives.dart:307:48)
#1      Runtime.execute (package:dart_eval/src/eval/runtime/runtime.dart:620:12)
#2      Runtime.executeLib (package:dart_eval/src/eval/runtime/runtime.dart:608:12)
```

**Comment by taosimple** [comment]
Change `primitives.dart` from: 
```
@override
  void run(Runtime runtime) {
    runtime.frame[_reg] = (runtime.frame[_reg] as $Value).$value;
  }
```
to:
```
  @override
  void run(Runtime runtime) {
    if(runtime.frame[_reg] is $Value){
      runtime.frame[_reg] = (runtime.frame[_reg] as $Value).$value;
    }
  }
```
eval success !

**Comment by ethanblake4** [comment]
Fixed in v0.6.3
